pkg/larker/larker.go

```diff
@@ -112,3 +124,98 @@ func (larker *Larker) Main(ctx context.Context, source string) (string, error) {
 
 	return formattedYaml, nil
 }
+
+func (larker *Larker) Hook(
+	ctx context.Context,
+	source string,
+	name string,
+	arguments []interface{},
+) (*HookResult, error) {
+	if name == "" {
+		return nil, fmt.Errorf("%w: empty hook name specified", ErrSanity)
+	}
+
+	if len(arguments) != 1 {
```

Comment:
It shouldn't matter. We'll decide it on the server side.

Reply:
Changed in https://github.com/cirruslabs/cirrus-cli/pull/303/commits/5b990fe1f4a554bee975f65c2a7e8331da170b77.